web/src/components/messages/loki-error.tsx

```diff
@@ -0,0 +1,200 @@
+      setLoading(true);
+
+      switch (type) {
+        case LokiInfo.Build:
+          setInfoName(t('Build info'));
+          getBuildInfo()
+            .then(data => setInfo(JSON.stringify(data, null, 2)))
+            .catch(err => {
+              setInfo(getHTTPErrorDetails(err));
+            })
+            .finally(() => {
+              setLoading(false);
+            });
+          break;
+        case LokiInfo.Limits:
+          setInfoName(t('Configuration limits'));
+          getLimits()
+            .then(data => setInfo(JSON.stringify(data, null, 2)))
+            .catch(err => {
+              setInfo(getHTTPErrorDetails(err));
+            })
+            .finally(() => {
+              setLoading(false);
+            });
+          break;
+        case LokiInfo.Metrics:
+          setInfoName(t('Metrics'));
+          getMetrics()
+            .then(data => setInfo(data))
+            .catch(err => {
+              setInfo(getHTTPErrorDetails(err));
+            })
+            .finally(() => {
+              setLoading(false);
+            });
+          break;
+        case LokiInfo.Hidden:
+        default:
+          setInfoName(undefined);
+          setInfo(undefined);
+          setLoading(false);
+          break;
+      }
+    },
+    [t]
+  );
+
+  React.useEffect(() => {
+    getLokiReady()
+      .then(() => {
+        setReady('');
+      })
+      .catch(err => {
+        setReady(getHTTPErrorDetails(err));
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+
+  return (
+    <div id="loki-error-container">
+      <EmptyState data-test="error-state">
+        <EmptyStateIcon className="loki-error-icon" icon={ExclamationCircleIcon} />
+        <Title headingLevel="h2" size="lg">
+          {title}
+        </Title>
+        <EmptyStateBody className="error-body">
+          <Text className="loki-error-message" component={TextVariants.p}>
+            {error}
+          </Text>
+          {_.isEmpty(ready) ? (
+            <TextContent className="error-text-content">
+              <Text component={TextVariants.p}>
+                {t('You may consider the following query changes to avoid this error')}
+              </Text>
+              {error.includes('max entries limit') && (
+                <>
+                  <Text component={TextVariants.blockquote}>
+                    {t('Reduce the Query Options -> limit to reduce the number of results')}
+                  </Text>
+                  <Text component={TextVariants.blockquote}>
+                    {t('Increase Loki "max_entries_limit_per_query" entry in configuration file')}
+                  </Text>
+                </>
+              )}
+              {(error.includes('deadline exceeded') || error.includes('maximum of series')) && (
+                <Text component={TextVariants.blockquote}>
+                  {t('Add some filters like Namespace or Name to reduce the number of results')}
```

Comment:
Sure, I will update the tips to mention indexes :+1: 

I would do the notice part in another PR since it will be a separate component and we don't need backend for that